### PR TITLE
Add automation (mostly inherited from Mock)

### PR DESCRIPTION
**Pull request by praiskup**

### CODE REVIEW

File: .packit.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      trigger: pull_request
+      packages:
+        - fedora-distro-aliases
+      targets:
+        - fedora-all
+
+  - <<: *pr_build
+    trigger: commit
+    owner: "@copr"
+    project: "copr"
```
Comment (FrostyX) [inline .packit.yaml]: I guess this is a leftover from copy-pasting. 

Currently, I have this Copr project 
https://copr.fedorainfracloud.org/coprs/frostyx/fedora-distro-aliases/ 

but we should probably create a new one in 
https://copr.fedorainfracloud.org/groups/g/rpm-software-management/coprs/ 
I don't have permissions though.

**Comment by FrostyX** [comment]
Than you very much for the PR, I am notoriously bad at adding these myself and always forget about it.

**Comment by mfocko** [comment]
> Than you very much for the PR, I am notoriously bad at adding these myself and always forget about it.

Since this project is just “starting”, you could probably use ruff right away (it gets harder once you have to fix a shitload of errors :grin:)

**Comment by praiskup** [comment]
> you could probably use ruff right away

But even for the new code ... the Ruff will evolve in the future, adding more checkers.  That would mean new warnings for the "future old code", and irrelevant in pull requests.  Differential linting is almost always a better option for pull-requests IMO.

File: .packit.yaml (hunk 1 above)
Comment (praiskup) [reply .packit.yaml]: No a c&p error, my mistake - I thought this lives in the `fedora-copr` namespace.

**Comment by praiskup** [comment]
I'll continue with this PR next week...   thank you for looking, folks!

File: .packit.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      trigger: pull_request
+      packages:
+        - fedora-distro-aliases
+      targets:
+        - fedora-all
+
+  - <<: *pr_build
+    trigger: commit
+    owner: "@rpm-software-management"
+    project: "fedora-distro-aliases"
```
Comment (FrostyX) [inline .packit.yaml:38]: I can see you were able to create the project, nice.
https://copr.fedorainfracloud.org/coprs/g/rpm-software-management/fedora-distro-aliases/

**Comment by FrostyX** [comment]
It looks like everything is done and this PR is finished, we are only waiting for the org ticket to be processed?

**Comment by praiskup** [comment]
I never know... do you think the automation needs to be merged first to start working? Then this is probably ready.

**Comment by praiskup** [comment]
For example, here it worked immediately.. https://github.com/fedora-copr/logdetective/pull/17

**Comment by praiskup** [comment]
Good to go, I think. PTAL

**Pull request merged**